var/spack/repos/builtin/packages/mash/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Mash(AutotoolsPackage):
+    """
+    Fast genome and metagenome distance estimation using MinHash.
+    """
+
+    homepage = "http://mash.readthedocs.org/"
+    url      = "https://github.com/marbl/Mash/archive/refs/tags/v2.3.tar.gz"
+
+    maintainers = ['marcusboden']
+
+    version('2.3', sha256='f96cf7305e010012c3debed966ac83ceecac0351dbbfeaa6cd7ad7f068d87fe1')
+
+    conflicts('%gcc@11:', msg='mash cannot be build with gcc >= 11')
+
+    depends_on("autoconf", type='build')
+    depends_on("automake", type='build')
+    depends_on("libtool", type='build')
+    depends_on("m4", type='build')
+    depends_on("capnproto", type='build')
```

Comment:
```suggestion
    depends_on("capnproto", type='build')
```

Is this only used during the build or should it be a link dep too?

Reply:
It works fine as a build dep, so I guess it's not needed at linking? At least the software runs fine after install. Or is there something else I should check?

Reply:
That's because it is statically linked, but if there are no static libs for capnproto, then the required to have `type=('link')` at least. I'd suggest to just drop `type='build'` here too, since you may not control what type of libraries capnproto builds.

Reply:
Ah, thanks for the explanation!